File: C#/백준/Bronze/10480. Oddities/Oddities.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            for (int i =0; i<a; i++){
                int b = int.Parse(Console.ReadLine());
                if(b%2==0){
                    Console.WriteLine($"{b} is even");
                }
                else{
                    Console.WriteLine($"{b} is odd");
                }
            }
        }
    }
}

File: Java/백준/Bronze/10480. Oddities/Oddities.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        for (int i=0; i<a; i++){
            int b = in.nextInt();
            if(b%2==0){
                System.out.printf("%d is even\n",b);
            }
            else{
                System.out.printf("%d is odd\n",b);
            }
        }
    }
}